### PR TITLE
removed unneeded imports

### PR DESCRIPTION
**Pull request by SamMarkowitz**
Signed-off-by: Sam Markowitz <sam.markowitz@hp.com>

### CODE REVIEW

**Comment by oritstone** [comment]
@SamMarkowitz is it ready for review & merge?

**Comment by SamMarkowitz** [comment]
@oritstone - yep. it's ready.

File: app/slang-content/examples/divide/master_divider.sl [hunk 1]
```diff
@@ -15,7 +15,7 @@ flow:
   workflow:
```
Comment (oritstone) [inline app/slang-content/examples/divide/master_divider.sl]: Remove unneeded import above

**Comment by oritstone** [comment]
@SamMarkowitz you can fix 1 comment above and then merge

**Pull request merged**